.gitignore

```diff
@@ -1 +1,6 @@
 /vendor
+.phpunit.result.cache
+Test/Unit
```

Comment:
Hmmm why ignore Test/Unit?

Reply:
It was created when running phpunit so I assumed it was a build artefact